doc/developer-guides/hld/ivshmem-hld.rst

```diff
@@ -0,0 +1,97 @@
+.. _ivshmem-hld:
+
+ACRN Shared Memory Based Inter-VM Communication
+###############################################
+
+To support inter virtual machines communication based on shared memory
+mechanism, ACRN device model/hypervisor emulate a virtual PCI device
+(Called ivshmem device) to expose the shared memory base address and
+size. Currently, ACRN only supports post-launched to post-launched
+communication via ivshmem device. In future, ACRN will supports
+pre-launched VM inter-VM communication.
```

Comment:
Remove "Currently, ACRN only supports post-launched to post-launched communication via ivshmem device. In future, ACRN will supports pre-launched VM inter-VM communication."
We can talk it later, not at the beginning.